README.md

```diff
@@ -12,6 +12,7 @@ An extended abstract of this work (https://arxiv.org/abs/2011.04789) appears in
 This package requires python>=3.8. Install the dependencies with
 
  - xargs -L 1 pip install < requirements.txt
+ - pip3 install pyope
```

Comment:
Did you try running your code with the version of pyope in third-party/? I have an open PR to remove it in favor of the pypi version, but for this PR, if you could use the third-party version, it would be nicer.